tests/utils/upgrade.go

```diff
@@ -29,9 +31,9 @@ import (
 	. "github.com/onsi/gomega"    // nolint
 )
 
-// EnableOnlineUpgradeForInstanceManager creates the operator namespace and enables tho online upgrade for
+// CreateOperatorConfigurationMap creates the operator namespace and enables/disable the online upgrade for
 // the instance manager
-func EnableOnlineUpgradeForInstanceManager(pgOperatorNamespace, configName string, env *TestingEnvironment) {
+func CreateOperatorConfigurationMap(pgOperatorNamespace, configName string, isOnline bool, env *TestingEnvironment) {
 	By("creating operator namespace", func() {
```

Comment:
IMO we could make the signature clearer. If this is meant to be more widely used, could take a map.
If only the online/rollout upgrade, we could have a constant, say `utils.EnableOnlineUpgrades`

Reply:
When calling functions with boolean values, always obscure...